src/App.js

```diff
@@ -42,7 +42,7 @@ function App() {
         <Route exact path="/" component={PetsList} />
         <Route
           exact
-          path="/petDetails/:id"
+          path="/petDetails/:animalId"
```

Comment:
i know this part didn't change, but can we just make this "pets/:animalId"?

Also for now, this should accept a status as a query param. This is not for any fault of yours, but this call is deathly slow without a "status" on the first load.

I have a note to add some work to figure out what is slowly up these calls, but have not had a chance to add it.

Reply:
changes to pets

The api url always has status=adoptable in it for now with the ability to accept any other status. 

const GetPets = (endPoint = "/hub/pets", status = "adoptable", petType = "") =>
  axios
    .get(
      `${getValue("apiRoot")}${endPoint}?status=${status}${

Is this what you are referring to?

Reply:
oh, sorry, i didn't notice that, yes, exactly, this would potentially become problematic (maybe just unneccessary) in the future if we are able to clean this up just to use id. Thanks for connected the dots.